cinq_auditor_iam/__init__.py

```diff
@@ -53,13 +54,36 @@ def run(self, *args, **kwargs):
             Account.enabled == 1,
             Account.account_type == AccountTypes.AWS
         )
+        self.manage_policies(accounts)
+        self.update_role_timeouts(accounts)
+
+    def update_role_timeouts(self, accounts):
+        if not accounts:
+            return
+        timeout_in_seconds = self.dbconfig.get('role_timeout_in_hours', self.ns, 8) * 60 * 60
+        for account in accounts:
+            sess = get_aws_session(account)
+            iam = sess.client('iam')
+            try:
+                role_list = iam.list_roles()['Roles']
+                for role in role_list:
```

Comment:
Should add a check here inside the loop to check if the MaxSessionDuration is already set to the requested value, and if not, just skip it so that we don't perform what amounts to noop API calls